volatility3/framework/layers/physical.py

```diff
@@ -118,7 +118,7 @@ def maximum_address(self) -> int:
         with self._lock:
             orig = self._file.tell()
             self._file.seek(0, 2)
-            self._size = self._file.tell()
+            self._size = self._file.tell() - 1
```

Comment:
Thanks, since this is no longer the size, could we change this variable name to _maximum_address please?

Reply:
name changed